roles/ansible-tdp-common-actions/templates/ansible-hosts.j2

```diff
@@ -11,6 +11,9 @@ edge-01
 [zk]
 master-0[1:3]
 
+[zk_kafka]
```

Comment:
What is this group ? In the README, the dedicated ZK for Kafka is on the `zk` host.

Reply:
It won't be needed, this fix is wrong as told by @Nuttymoon [here](https://github.com/TOSIT-IO/tdp-getting-started/pull/136#issuecomment-1190096329)

I will open a new PR with the correct fix

Reply:
Yes I read my mails after I comment this PR.